packages/store/types/tsconfig.json

```diff
@@ -20,9 +20,9 @@
     "baseUrl": ".",
     "paths": {
       "@angular/*": ["../../../node_modules/@angular/*"],
-      "@ngxs/store/operators": ["../../../packages/store/operators/src"],
-      "@ngxs/store/internals": ["../../../packages/store/internals/src"],
-      "@ngxs/*": ["../../../packages/*"]
+      "@ngxs/store/operators": ["../../../@ngxs/store/operators/index.d.ts"],
+      "@ngxs/store/internals": ["../../../@ngxs/store/internals/index.d.ts"],
```

Comment:
Is this how we simulate the `strict` compile options?
This does introduce the dependency that these tests can only be run after the build is done. Are the scripts adjusted accordingly?

Reply:
Unfortunately, otherwise we cannot verify the reality of the work

Reply:
This does introduce the dependency that these tests can only be run after the build is done. Are the scripts adjusted accordingly?

Reply:
These tests should be run after assembly! Since they guarantee that the real API is built under TS